howso/direct/core.py

```diff
@@ -372,14 +318,21 @@ def create_trainee(self, trainee_id: str) -> Union[Dict, None]:
         dict
             A dict containing the name of the trainee that was created.
         """
-        fname = self.trainee_template_fname.split('.')
-        return self._execute("create_trainee", {
-            "trainee": trainee_id,
-            "filepath": f"{self.trainee_template_path}/",
-            "trainee_template_filename": fname[0],
-            "file_extension": fname[1],
-            "trainee_id": trainee_id
+        status = self.amlg.load_entity(
+            trainee_id,
+            str(self.howso_fully_qualified_path),
+            False,
+            True,
+            False,
+            False
+        )
+        self._execute(trainee_id, "initialize", {
+            "trainee_id": trainee_id,
+            "filepath": str(self.howso_path) + '/',
         })
+        if not status.loaded:
+            raise HowsoError("Error loading the Trainee.")
+        return trainee_id
```

Comment:
Need to also update the return type of this method.

Reply:
I'll just update the return value to match the return type.